README.md

```diff
@@ -48,13 +48,42 @@ Git Proxy is built with a developer-first mindset. By presenting simple-to-follo
 To install Git Proxy, use the [npm](https://www.npmjs.com/) package manager:
 
 ```bash
-$ npm install @finos/git-proxy
+$ npm install -g @finos/git-proxy
+$ git-proxy
 ```
 
 To install a specific version of Git Proxy, append the version to the end of the `install` command:
 
 ```bash
-$ npm install @finos/git-proxy@1.0.0
+$ npm install -g @finos/git-proxy@1.0.0
+```
+
+You can also run directly using `npx`:
+
+```bash
+$ npx @finos/git-proxy
+```
+
+## Configuration
+By default, git-proxy ships with a [default configuration](./proxy.config.json) for demonstration purposes. In most environments, this should be overridden by your user-specific values.
+
+To set your own values, create a `proxy.config.json` in the current working directory. This will be loaded when you execute `git-proxy` if present.
+
+If you wish to specify a different file location to use as configuration, use the `-c/--config` command-line argument:
+
+```bash
+$ git-proxy --config /etc/gitproxy/config.json
+# With npx
+$ npx -- git-proxy --config /etc/gitproxy/config.json 
+```
+
```

Comment:
We should probably add some high-level information about the core parts of the configuration and what they do here. Happy to help, but we can also open this as a separate issue.

@coopernetes?

Reply:
@coopernetes - there are probably of `npm `libraries out there.

https://github.com/adobe/jsonschema2md at first glance?

Reply:
Gave this a go and it likely doesn't meet our needs. A number of unresolved issues that I ran into immediately when trying to generate docs based on the new schema:

https://github.com/adobe/jsonschema2md/issues/531
https://github.com/adobe/jsonschema2md/issues/392
https://github.com/adobe/jsonschema2md/issues/276